es-space-mgmt/main/default/lwc/spaceDesigner/spaceDesigner.js

```diff
@@ -1,48 +1,76 @@
 import { LightningElement, api, wire } from 'lwc';
 import { ShowToastEvent } from 'lightning/platformShowToastEvent';
-import { CurrentPageReference } from 'lightning/navigation';
-import { registerListener, unregisterAllListeners, fireEvent } from 'c/pubsub';
+
+import TILE_SELECTION_MC from '@salesforce/messageChannel/Tile_Selection__c';
+import FLOW_STATUS_CHANGE_MC from '@salesforce/messageChannel/Flow_Status_Change__c';
+import {
+    subscribe,
+    unsubscribe,
+    APPLICATION_SCOPE,
+    MessageContext,
+    publish
+} from 'lightning/messageService';
 
 export default class SpaceDesigner extends LightningElement {
     /*
      *   Component coordinates event comms between Aura-based parent component
-     *   and LWC-based siblings. Uses pubsub in place of custom Aura application event.
+     *   and LWC-based siblings. Uses LMS in place of custom Aura application event.
      *   TO DO: Replace Aura parent component when support for LWC flow screens available.
      */
     flowStarted = false;
-    @wire(CurrentPageReference) pageRef;
+    subscription = null;
+
+    @wire(MessageContext)
+    messageContext;
+
+    subscribeToMessageChannel() {
+        if (!this.subscription) {
```

Comment:
Same comment as `customerList.js` about that check.

Reply:
Updated